### DISCUSSION TITLE
Granfana folder

### DISCUSSION THREAD

**Discussion by Iatbzh** [grafana/grafana#45022] [Platform]
Hello, since grafana8.3, only multiple folders can be created, and folders cannot be created under folders, so the dashboard is very messy. I hope it can be restored and can be organized.

**Comment by torkelo** [comment]
nested folders have never been supported

>  only multiple folders can be created

?

**Reply by Iatbzh** [reply]
So can I do such an update

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
